app/src/main/res/layout/videoplayer_activity.xml

```diff
@@ -33,27 +33,27 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_margin="8dp"
-            android:background="@drawable/overlay_button_circle_background"
+            android:background="@drawable/md_transparent"
```

Comment:
Instead of explicitly making the background transparent, you can just remove the attribute.

Reply:
I tried to do this, but the background default is not transparent. I guess this goes through the configuration of the theme.